packages/js/product-editor/src/components/editor/editor.tsx

```diff
@@ -101,14 +100,6 @@ export function Editor( {
 							</ValidationProvider>
 						</SlotFillProvider>
 					</ShortcutProvider>
-					{ /* We put Footer here instead of in InterfaceSkeleton because Footer uses
-					WooFooterItem to actually render in the WooFooterItem.Slot defined by
-					WooCommerce Admin. And, we need to put it outside of the SlotFillProvider
-					we create in this component. */ }
-					<Footer
-						productId={ product.id }
-						productType={ product.type }
-					/>
```

Comment:
I removed this, as it wasn't Editor specific. This now lives in the `product-page` and `product-variation-page` respectively.
This way it was easier to assign certain footer elements for certain pages. As we only wanted the variation switcher on the variation page and the CES Feedback bar on the product page.